src/Cli/Microsoft.TemplateEngine.Cli/Help/HelpBuilder.cs

```diff
@@ -0,0 +1,533 @@
+                }
+                secondColumnWidth = windowWidth - firstColumnWidth - Indent.Length - Indent.Length;
+            }
+
+            for (var i = 0; i < items.Count; i++)
+            {
+                var helpItem = items[i];
+                IEnumerable<string> firstColumnParts = WrapText(helpItem.FirstColumnText, firstColumnWidth);
+                IEnumerable<string> secondColumnParts = WrapText(helpItem.SecondColumnText, secondColumnWidth);
+
+                foreach (var (first, second) in ZipWithEmpty(firstColumnParts, secondColumnParts))
+                {
+                    context.Output.Write($"{Indent}{first}");
+                    if (!string.IsNullOrWhiteSpace(second))
+                    {
+                        int padSize = firstColumnWidth - first.Length;
+                        string padding = "";
+                        if (padSize > 0)
+                        {
+                            padding = new string(' ', padSize);
+                        }
+
+                        context.Output.Write($"{padding}{Indent}{second}");
+                    }
+
+                    context.Output.WriteLine();
+                }
+            }
+
+            static IEnumerable<(string, string)> ZipWithEmpty(IEnumerable<string> first, IEnumerable<string> second)
+            {
+                using var enum1 = first.GetEnumerator();
+                using var enum2 = second.GetEnumerator();
+                bool hasFirst = false, hasSecond = false;
+                while ((hasFirst = enum1.MoveNext()) | (hasSecond = enum2.MoveNext()))
+                {
+                    yield return (hasFirst ? enum1.Current : "", hasSecond ? enum2.Current : "");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets a help item for the specified symbol.
+        /// </summary>
+        /// <param name="symbol">The symbol to get a help item for.</param>
+        /// <param name="context">The help context.</param>
+        public TwoColumnHelpRow GetTwoColumnRow(
+            Symbol symbol,
+            HelpContext context)
+        {
+            if (symbol is null)
+            {
+                throw new ArgumentNullException(nameof(symbol));
+            }
+
+            Customization? customization = null;
+
+            if (_customizationsBySymbol is { })
+            {
+                _customizationsBySymbol.TryGetValue(symbol, out customization);
+            }
+
+            if (symbol is Option or Command)
+            {
+                return GetOptionOrCommandRow();
+            }
+            else if (symbol is Argument argument)
+            {
+                return GetCommandArgumentRow(argument);
+            }
+            else
+            {
+                throw new NotSupportedException($"Symbol type {symbol.GetType()} is not supported.");
+            }
```

Comment:
nit: If-Else is redundant.
```suggestion
            if (symbol is Option or Command)
            {
                return GetOptionOrCommandRow();
            }

            if (symbol is Argument argument)
            {
                return GetCommandArgumentRow(argument);
            }

            throw new NotSupportedException($"Symbol type {symbol.GetType()} is not supported.");
```